Automattic/wp-calypso
# Reader: Add search-stream header for site-results-in-search

This is an extraction of the header for site-results-in-search project that was created as part of this pr: https://github.com/Automattic/wp-calypso/pull/13198

#### wide version:
![results-w-sites-wide](https://cloud.githubusercontent.com/assets/4924246/25633390/c5660b30-2f2b-11e7-9cba-843f9efa1f7d.png)

#### narrow version:
![results-w-sites-mobile-1](https://cloud.githubusercontent.com/assets/4924246/25633394/c939a74e-2f2b-11e7-87f5-97cc8063355f.png)

The header should be two columns and then turn into the section navs at smaller width

TODO:
- [x] add in the styles
- [x] ~~add to devdocs~~

## To Test:
~go to http://calypso.localhost:3000/devdocs/blocks/search-stream-header and verify the header looks/behaves as expected~
